MediaRecorder.html

```diff
@@ -469,12 +510,27 @@
         </dl>
       </section>
     </div>
+    <section>
+      <h2>Data handling</h2>
+        <p><dfn data-dfn-type="dfn"
+        data-lt="fire a blob event">Firing a blob event</dfn> with a
```

Comment:
Optional nit: this is consistent with "Firing an error event", but if you think that'll go away "To fire a blob event named *e* at *target* with a `Blob` *blob*, fire an event..." would be more pleasing.